dbsnp-importer/src/main/java/uk/ac/ebi/eva/dbsnpimporter/models/SubSnpCoreFields.java

```diff
@@ -236,6 +270,14 @@ public Orientation getHgvsTOrientation() {
         return hgvsTOrientation;
     }
 
+    public List<Map<String, String>> getGenotypeList() {
```

Comment:
Given that we are not storing the plain string, it could be simpler to call this method `getGenotypes` (also applies to related methods and variables)